lib/dynamoid/adapter.rb

```diff
@@ -19,7 +19,7 @@ def tables
       if !@tables_.value
         @tables_.swap{|value, args| benchmark('Cache Tables') {list_tables}}
       end
-      @tables_.value
+      @tables_.value || []
```

Comment:
Do you have any insight @adamthedeveloper? just trying to find the scenario to properly wrap a test case for it.

Thanks!